module_utils/network_lsr/nm/active_connection.py

```diff
@@ -21,6 +25,7 @@ def deactivate_active_connection(nm_ac, timeout, check_mode):
         return False
     if not check_mode:
         main_loop = client.get_mainloop(timeout)
+        # pylint: disable=logging-format-interpolation
         logging.debug(
```

Comment:
Seems to me that pylint is right here and instead of disabling the check, using `logging.debug("Deactivating %s ...", nm_ac.get_id())` would be the better choice here and for the other instances.

Reply:
I rewrote all of the calls to `logging` methods to use `%s`